com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -195,6 +195,9 @@ private static void OnBakeStarted()
             if (refVolAuthList.Length == 0)
                 return;
 
+            FindWorldBounds();
+            refVolAuthList = GameObject.FindObjectsOfType<ProbeReferenceVolumeAuthoring>();
```

Comment:
Why do we need to get the list twice? It can change between calls?

Reply:
Yeah it can get invalidated when loading other scenes